### DISCUSSION TITLE
dynamic slug pages

### DISCUSSION THREAD

**Discussion by nestle49** [nuxt/nuxt#24755] [Questions]
For example page 'About' have slug 'site/about/', but user inside admin panel redefine page slug and api give new slug 'site/about-us/'

Can I make it? How?

**Comment by gjssss** [comment]
Could you describe the question more clearly?

**Reply by nestle49** [reply]
The user can select a custom slug for the root pages in CRM, for example 'about' or 'about-us'. This parameter gives the api

How to properly implement this in Nuxt?

**Comment by OnlyWick** [comment]
See the documentation
- nuxt3: https://nuxt.com/docs/guide/directory-structure/pages#dynamic-routes
- nuxt2: https://v2.nuxt.com/docs/features/file-system-routing#dynamic-routes
